src/App.tsx

```diff
@@ -15,36 +24,90 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+function getPreperedGoods(
+  goods: Goods,
+  sortField: SortStatus | '',
+  reverseField: boolean,
+) {
+  const preparedGoods = [...goods];
+
+  if (typeof sortField !== 'undefined') {
```

Comment:
You can remove it because if the sort field is undefined it will be a default switch case

And to make it safe you can use the current sort field in the case when you don't need sorting